AnnoyancesFilter/sections/push-notifications.txt

```diff
@@ -121,6 +121,7 @@ youla.ru##.web_push_confirmation
 doviz.com,sinemalar.com###x-comet-dialog
 stalkermods.ru,ucoz.net##.pshsndr
 calciomercato.com###dialog-push-notification
+tokar.ua##.bouncing-icon
```

Comment:
To popups since it is always shown on the page (even though it is a link to their twitter).